tracks/python/exercises/isogram/mentoring.md

```diff
@@ -14,8 +14,8 @@ def is_isogram(string):
         ch = string[i]
 
         if ch.isalpha():
-		    if (s.find(ch) > -1):
-			    return False
+	    if (s.find(ch) > -1):
+	        return False
```

Comment:
This indentation should not have been reduced.

Reply:
Agreed.  

Sorry - I have been away from the keyboard for most of a month.